zpeclient.go

```diff
@@ -10,9 +10,16 @@ import (
 
 	"github.com/yahoo/athenz/clients/go/zms"
 	v1 "github.com/yahoo/k8s-athenz-syncer/pkg/apis/athenz/v1"
+	corev1 "k8s.io/api/core/v1"
 	"k8s.io/client-go/tools/cache"
 )
 
+const (
+	lastUpdateKeyField string = "latest_contact"
```

Comment:
We can get rid of the string type here. It will automatically get assigned to a string type.